lib/rspec/matchers/english_phrases.rb

```diff
@@ -0,0 +1,58 @@
+module RSpec
+  module Matchers
+    # Facilitates converting ruby objects to English phrases.
+    module EnglishPhrases
```

Comment:
I think I figured out what I dislike about `EnglishPhrases`...as a plural, it sounds like it's a list of phrases, whereas `EnglishPhrasing` reads more like it helps you create strings using English phrasing (which is more accurate to what it does).